src/journal-remote/journal-remote.c

```diff
@@ -188,7 +188,7 @@ static int open_output(Writer *w, const char* host) {
 
         r = journal_file_open_reliably(output,
                                        O_RDWR|O_CREAT, 0640,
-                                       arg_compress, arg_seal,
+                                       arg_compress, -1, arg_seal,
```

Comment:
i'd prefer not to pass non-casted negative values to unsigned parameters. Yes, C accepts that, but I think it's a bit misleading, because it hides the implicit cast done here and that on the receiving side you suddenly have a very large value and not a small one. please pass `(uint64_t) -1` or `UINT64_MAX` here, acknowledging the unsigned nature of this parameter and making the cast explicit